src/styles/home.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 100px;
+}
```

Comment:
- [ ] It would be better to make your application look like the design on the mobile screen size, please pay attention to screen sizes from `375px` above your mobile screen looks good from `410px`, please focus more on the mobile screen sizes this will make your design look more like the Behance design and look great on mobile devices, see video below 👍🏿 



https://user-images.githubusercontent.com/69638013/205116997-b495e4b7-db61-409e-a669-3aec5a145244.mp4